src/lib/Command.ts

```diff
@@ -9,14 +9,16 @@ export class Command {
     execute: (options: CommandExecuteParameters) => void
     data: SlashCommandBuilder
     args: CommandArg[] = []
+    cooldown?: Number //Seconds
 
     constructor(name: string, description: string, aliases: string[], execute: (options: CommandExecuteParameters) => void,
-        options?: CommandArg[]) {
+    cooldown: Number, options?: CommandArg[]) {
```

Comment:
Wrong type.. why didn't you catch me? "Number" vs "number"- /joking

Reply:
lol, typescript is not confusing at all /s